project-base/storefront/components/Pages/CategoryDetail/CategoryBestsellers/CategoryBestsellersListItem.tsx

```diff
@@ -0,0 +1,80 @@
+import { useDomainConfig } from 'hooks/useDomainConfig';
+import { twJoin } from 'tailwind-merge';
+
+type CategoryBestsellersListItemProps = {
+    product: ListedProductFragmentApi;
+    gtmProductListName: GtmProductListNameType;
+    listIndex: number;
+};
+
+const TEST_IDENTIFIER = 'pages-category-bestseller-item-';
+
+export const CategoryBestsellersListItem: FC<CategoryBestsellersListItemProps> = ({
+    product,
+    gtmProductListName,
+    listIndex,
+}) => {
+    const formatPrice = useFormatPrice();
+    const { url } = useDomainConfig();
+
+    return (
+        <div className="flex flex-wrap items-center gap-y-4 border-t border-greyLight py-4 first-of-type:border-0 lg:flex-nowrap lg:gap-5">
+            <div className="flex w-full items-center lg:w-7/12">
+                <div data-testid={TEST_IDENTIFIER + 'name'}>
+                    <ExtendedNextLink
+                        type="product"
+                        href={product.slug}
+                        className="flex items-center gap-5 font-bold no-underline"
+                        onClick={() => onGtmProductClickEventHandler(product, gtmProductListName, listIndex, url)}
+                    >
+                        <Image
+                            dataTestId={TEST_IDENTIFIER + 'image'}
+                            image={product.mainImage}
+                            type="list"
+                            alt={product.fullName}
+                            className="max-w-[80px]"
+                        />
+                        <span>{product.fullName}</span>
+                    </ExtendedNextLink>
+                </div>
+            </div>
+
+            <div className="w-4/6 lg:w-3/12 lg:text-center">
+                <span
+                    className={twJoin(
+                        product.availability.status === AvailabilityStatusEnumApi.InStockApi && 'text-inStock',
+                        product.availability.status === AvailabilityStatusEnumApi.OutOfStockApi && 'text-red ',
+                    )}
+                >
+                    {product.availability.name}
+                </span>
+
+                <ProductAvailableStoresCount
+                    isMainVariant={product.isMainVariant}
+                    availableStoresCount={product.availableStoresCount}
+                />
+                <ProductExposedStoresCount
+                    isMainVariant={product.isMainVariant}
+                    exposedStoresCount={product.exposedStoresCount}
+                    className="text-inStock"
+                />
+            </div>
+
+            <div className="w-2/6 text-right lg:w-2/12" data-testid={TEST_IDENTIFIER + 'totalprice'}>
+                <div>
+                    <div className="text-h6 font-bold leading-5 text-primary" data-testid={TEST_IDENTIFIER + 'price'}>
+                        {formatPrice(product.price.priceWithVat)}
+                    </div>
+                </div>
+            </div>
```

Comment:
I understand that until it's being decided if remove or not remove `data-testid` in scenarios where aren't those values used you want to follow the convention but is it really necessary to include them here in new components?

Reply:
Let's refactor this to usage only for one `div` no need for those three.

Reply:
Removed data test IDs and also those redundant divs